demo/OpenVINO/cpp/yolox_openvino.cpp

```diff
@@ -234,7 +236,7 @@ static void decode_outputs(const float* prob, std::vector<Object>& objects, floa
         std::vector<int> strides = {8, 16, 32};
         std::vector<GridAndStride> grid_strides;
 
-        generate_grids_and_stride(INPUT_W, strides, grid_strides);
+        generate_grids_and_stride(INPUT_W, INPUT_H, strides, grid_strides);
```

Comment:
This line is inconsistent with the definition of generate_grids_and_stride, which takes "input_h, input_w" order.

Reply:
@ruinmessi I have changed input_h and input_w in the function "generate_grids_and_stride" and tested the new submition.